Source/Scene/PolylineCollection.js

```diff
@@ -1013,6 +1013,14 @@ define([
         }
     }
 
+    function replacer(key, value) {
+        if(defined(value) && value.hasOwnProperty('id')) {
```

Comment:
Does `value instanceof Texture` not work here? We should only do this for textures.